storeProjectHW/storeProjectHW/InfoProductViewController.swift

```diff
@@ -147,17 +148,34 @@ final class InfoProductViewController: UIViewController {
     // MARK: - Public Properties
     var images = Constants.images
     var productInfo = Constants.macBook
+    var productTag = Constants.prodTag
     
     // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
         initMethods()
+        myScrollView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTap)))
     }
     // MARK: - Private objc methods
-    @objc private func rightHandAction() {
-        
-    }
     
+    @objc private func handleTap(gest: UIGestureRecognizer) {
+        let webProductViewController = WebViewController()
+        switch productTag {
+        case 0:
+            webProductViewController.urlOfContent = "https://re-store.ru/catalog/P058-109-15/"
```

Comment:
```suggestion
            webProductViewController.urlOfContent = ""
```
Все текстовки в константы